addons/beehave/nodes/beehave_tree.gd

```diff
@@ -1,3 +1,10 @@
 ## Base class for all parts of the behaviour tree.
 class_name BeehaveTree extends Node
 @icon("../icons/icon.svg")
+
+
+enum {
```

Comment:
What is the rationale behind moving this from `BeehaveNode` to `BeehaveTree`?

Reply:
This enum was being defined both in `BeehaveNode` and `BeehaveRoot`. I moved to BeehaveTree since both classes extend from it.